libs/components/ag-grid/src/lib/modules/ag-grid/fixtures/ag-grid-data-manager.component.fixture.ts

```diff
@@ -83,7 +83,7 @@ export class SkyAgGridDataManagerFixtureComponent implements OnInit {
   }
 
   public ngOnInit(): void {
-    this.gridOptions = this.#gridService.getEditableGridOptions({
+    this.gridOptions = this.#gridService.getGridOptions({
```

Comment:
This probably always should've been `getGridOptions` — we remove `rowSelection` in `getEditableGridOptions`, and it now does nothing when calling `selectAll` when `rowSelection` is not configured.